### PR TITLE
Display original_file's tech metadata on FileSet show page

### LINKED ISSUE #63: Display technical metadata in FileSet properties.

### PR DESCRIPTION
**Pull request by escowles**
Fixes #63

### CODE REVIEW

File: app/models/file_metadata.rb [hunk 1]
```diff
@@ -10,16 +10,23 @@ class FileMetadata < Valkyrie::Resource
...
 
   def derivative?
-    use.include?(Valkyrie::Vocab::PCDMUse.ServiceFile)
+    use_uris.include?(Valkyrie::Vocab::PCDMUse.ServiceFile)
   end
+
+  private
+
+    def use_uris
+      use.map { |u| u.is_a?(Hash) ? u[:@id] : u }
```
Comment (tpendragon) [inline app/models/file_metadata.rb]: Why's this necessary? Valkyrie should support mapping back/forth to RDF::URI.

Comment (escowles) [reply app/models/file_metadata.rb]: When the use values are coming back from Solr, they come back as hashes like `{"@id":"http://pcdm.org/use#OriginalFile"}` — I'll try to figure out why Valkyrie isn't converting them to URIs.

Comment (escowles) [reply app/models/file_metadata.rb]: Oh, it's probably because the use values are nested objects inside the FileMetadata nested objects.

**Comment by escowles** [comment]
I've updated this now that https://github.com/samvera-labs/valkyrie/pull/206 fixed the nested URI issue.

**Pull request merged**